cypress.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = defineConfig({
     toConsole: true,
   },
   requestTimeout: 60000,
-  retries: 0,
+  retries: 4,
```

Comment:
Nice. wrapping the Beforeall in an it block should hopefully help the tests run more smoothly.

I am okay with this retry level logic as it will re-run an individual describe block, however we should have some caution as we proceed as certain retry logic can be destructive for the team. Previously I was on a team where e2e tests were re-run from the pipeline level up to 3 times if they failed. This lead to e2e tests that could take an hour to run and the tests to be written with a little less care, as it did not matter as much if an individual test was a little flakey.

Reply:
yeah this used to be set to 4 for the longest time, I reduced it to 0 maybe a month or two ago because I thought these were more consistent, but I guess I was wrong by moving it down to 0.  I personally would like all the retries to be 0 if possible since it would mean our tests are resilient, but I guess the state of e2e tests makes that goal hard.